### PR TITLE
Add NVM info and clarify Apple Silicon support

### PR DESCRIPTION
**Pull request by BytesGuy**
# Description
Add NVM info and clarify Apple Silicon support

# Reasons
CPE-197

### CODE REVIEW

**Comment by BytesGuy** [comment]
Also added Xcode 13 Beta 4 details

File: jekyll/_cci2/testing-ios.md [hunk 1]
```diff
@@ -43,18 +43,16 @@ To read about our customer support policy regarding beta images, please check ou
...
 
-It is possible to build Apple Silicon/Universal binaries using the Xcode `12.0.0-beta` image as Apple provides both the Intel (`x86_64`) and Apple Silicon (`arm64`) toolchains in this release. Cross-compiling Apple Silicon binaries on Intel hosts has an additional overhead and as a result compilation times will be longer than native compilation for Intel.
-
-Running or testing Apple Silicon apps natively is currently not possible as CircleCI build hosts are Intel-based Macs. Binaries will need to be exported as [artifacts](https://circleci.com/docs/2.0/artifacts/) for testing apps locally.
+Running or testing Apple Silicon apps natively is currently not possible as CircleCI build hosts are Intel-based Macs. Binaries will need to be exported as [artifacts](https://circleci.com/docs/2.0/artifacts/) for testing apps locally. Alternatively, [CircleCI Runner](https://circleci.com/docs/2.0/runner-overview/#supported) can also be used to run jobs natively on Apple Silicon.
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-ios.md:48]:
```suggestion
Running or testing Apple Silicon apps natively is currently not possible as CircleCI build hosts are Intel-based Macs. Binaries will need to be exported as [artifacts](https://circleci.com/docs/2.0/artifacts/) for testing apps locally. Alternatively, [CircleCI runner](https://circleci.com/docs/2.0/runner-overview/#supported) can also be used to run jobs natively on Apple Silicon.
```

File: jekyll/_cci2/testing-ios.md [hunk 2]
```diff
@@ -334,6 +332,53 @@ steps:
...
+
+To install a specific version of NodeJS and use it:
+
+```yaml
+# ...
+steps:
+  - run: nvm install 12.22.3 && nvm alias default 12.22.3
+```
+
+These images are also compatiable with the official [CircleCI Node Orb](https://circleci.com/developer/orbs/orb/circleci/node) which helps to manage your NodeJS installation along with caching packages.
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-ios.md:371]:
```suggestion
These images are also compatiable with the official [CircleCI Node orb](https://circleci.com/developer/orbs/orb/circleci/node) which helps to manage your NodeJS installation along with caching packages.
```

File: jekyll/_cci2/testing-ios.md [hunk 3]
```diff
@@ -334,6 +332,53 @@ steps:
...
+
+### Images using Xcode 12.5 and earlier
+{: #images-using-xcode-125-and-earlier }
+
+These images come with at least one version of NodeJS installed directly via `brew`.
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-ios.md:376]:
```suggestion
These images come with at least one version of NodeJS installed directly using `brew`.
```

File: jekyll/_cci2/testing-ios.md [hunk 4]
```diff
@@ -334,6 +332,53 @@ steps:
...
+
+Version information for the installed NodeJS versions can be found in [the software manifests for the image](#supported-xcode-versions)]
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-ios.md:378]:
```suggestion
Version information for the installed NodeJS versions can be found in [the software manifests for the image](#supported-xcode-versions)].
```

File: jekyll/_cci2/testing-ios.md [hunk 5]
```diff
@@ -334,6 +332,53 @@ steps:
...
+
+These images are also compatiable with the official [CircleCI Node Orb](https://circleci.com/developer/orbs/orb/circleci/node) which helps to manage your NodeJS installation, by installing `nvm`, along with caching packages.
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-ios.md:380]:
```suggestion
These images are also compatiable with the official [CircleCI Node orb](https://circleci.com/developer/orbs/orb/circleci/node) which helps to manage your NodeJS installation, by installing `nvm`, along with caching packages.
```

**Review by rosieyohannan — COMMENTED** [review]
Just a few minor changes and fixes then good to go, thanks @BytesGuy!

**Review by rosieyohannan — APPROVED** [review]
Thanks @BytesGuy!

**Pull request merged**